Orange/data/table.py

```diff
@@ -1202,6 +1205,7 @@ def _filter_values_indicators(self, filter):
                 sel = np.vectorize(f)(col)
             elif isinstance(f, (data_filter.FilterContinuous,
                                 data_filter.FilterString)):
+                col = get_str_column(col, isinstance(f, data_filter.FilterString))
```

Comment:
We measure cyclomatic complexity to make the code more, not less readable. :)

Find another way to decrease the complexity. Cheating with dictionaries typically makes the code slower or more complex, but in this case dictionaries are natural and fast. You can prepare a dictionary (as a class attribute, not a local variable)

```
    operators = {
        f.Equal: operator.eq,
        f.NotEqual: operator.neq,
        f.Less: operator.le,
        ...
    }
```

Then you can replace most if-s with a single `col = ops[f.oper](col, fmin)`.

Perhaps this does not work for some reason. In this case, find another solution to simplify the function. Extracting individual ifs to local functions is not a proper solution to high cyclomatic complexity (except when it makes the code actually better).

You can also extract the code the is within the for loop into a separate function. This would make sense since this is the code that applies the filter to a selection. If this is still not enough, make a separate function for each filter type (elifs in this block of code), and extract the function that prepares the initial `sel` column.